components/ui/stat-box.tsx

```diff
@@ -0,0 +1,8 @@
+export const Box = (props: any) => {
+    return (
+        <div className='mx-auto flex w-5/12 items-center justify-center rounded-lg bg-slate-800 object-center'>
```

Comment:
width of 5/12 on small screens make the boxes look skinny and weird. Resolve the previous comment and for this lets hard code the width to something (perhaps one width for each screen size)